GitHub Actions: Re-enable jacoco coverage report

When updating `org.jetbrains.intellij` in 8145a8605a969753540862b8fc7f6a27fd10923d (#59) the jacoco report generation started failing with the errors
> Caused by: : Error while creating report
> [...]
> Caused by: java.io.IOException: Error while analyzing /home/runner/work/ddev-intellij-plugin/ddev-intellij-plugin/build/classes/java/main/de/php_perfect/intellij/ddev/statusBar/DdevStatusBarWidgetFactoryImpl.class.
> [...]
> Caused by: java.lang.IllegalStateException: Can't add different class with same name: de/php_perfect/intellij/ddev/statusBar/DdevStatusBarWidgetFactoryImpl

This doesn't feel like an error on our side, so the commit ea6955a6bda37bf51e080b3bc568ef0b0fe59623 in (#59) should be reverted as soon as an update is available that fixes the issue.

- - -

Additionally the SonarCloud workflow should be re-enabled again. It keeps failing with OutOfMemory errors

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -29,7 +29,7 @@ jobs:
       - name: Run tests
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: test --stacktrace --info
+          arguments: test --stacktrace --info jacocoTestReport
           cache-read-only: ${{ github.ref != 'refs/heads/main' }}
 
       - name: Upload Test Report
@@ -37,3 +37,10 @@ jobs:
         with:
           name: test-report-${{ matrix.os }}
           path: build/reports/tests/test
+
+      - name: Upload HTML test coverage report
+        if: matrix.os == 'ubuntu-latest'
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report
+          path: build/reports/jacoco/test/html

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -2,7 +2,7 @@ import org.jetbrains.intellij.tasks.ListProductsReleasesTask
 
 plugins {
     id("org.jetbrains.changelog") version "1.3.1"
-    id("org.jetbrains.intellij") version "1.7.0"
+    id("org.jetbrains.intellij") version "1.8.0"
     java
 
     id("org.sonarqube") version "3.4.0.2513"

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -90,7 +90,7 @@ tasks {
 
 /* SonarCloud */
 tasks.sonarqube {
-    dependsOn(tasks.build)
+    dependsOn(tasks.build, tasks.jacocoTestReport)
 
     sonarqube {
         properties {

